components/Navbar.js

```diff
@@ -21,8 +21,9 @@ export default function Navbar() {
       label: "Contact",
     },
   ];
-  const QUICKET_LINK = "#";
+  const QUICKET_LINK = "https://qkt.io/w31vDE";
   const SESSIONIZE_LINK = "https://sessionize.com/pycon-uganda";
+  const TICKET_LABEL = "Get Your Ticket";
```

Comment:
About this label...do ya'll think we are good with the wording `Get Your Ticket`

Reply:
Personally, I'm fine with just `Get Ticket`

Reply:
@basiima this is noted and resolved via [ce73449](https://github.com/PyconUg/ug.pycon.org/pull/45/commits/ce73449799c0f1de000c11cbf4ad49827ad82520)